pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -195,13 +200,87 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
             <InfoCard
               title=""
               content={
-                <RatingIndicatorTemplate
-                  layoutClassName={styles.ratingIndicatorContainer}
-                  component={_getComponent.data}
-                />
+                <>
+                  {_getComponent.data.embedded?.rating ? (
+                    <RatingIndicatorTemplate
+                      layoutClassName={styles.ratingIndicatorContainer}
+                      maxRating={_getComponent.data.embedded?.rating?.maxRating}
+                      rating={_getComponent.data.embedded?.rating?.rating}
+                    />
+                  ) : (
+                    <div className={styles.noRatingStyle}>{t("No rating available")}</div>
+                  )}
+                  <span
+                    onClick={() => {
+                      show();
+                    }}
+                    className={styles.link}
+                  >
+                    <Link icon={<ArrowRightIcon />} iconAlign="start">
+                      Rating
+                    </Link>
+                  </span>
+                </>
               }
               layoutClassName={styles.infoCard}
             />
+            {isVisible && (
+              <div className={styles.overlay}>
+                <NotificationPopUp
+                  {...{ hide, isVisible }}
+                  title={"Rating"}
+                  description={
+                    <>
+                      {_getComponent.data.embedded?.rating?.rating ? (
+                        <p>{`${_getComponent.data.embedded?.rating?.rating}/${_getComponent.data.embedded?.rating?.maxRating}`}</p>
+                      ) : (
+                        <p>{t("No rating available")}</p>
```

Comment:
`p` tags are reserved to contain paragraphs of text (and even better used whenever in the context of e.g. an article with headings and such).

In this context however, we're really not displaying any type of content, therefore a `span` would be better suitable.